diesel_cli/tests/support/command.rs

```diff
@@ -66,6 +66,15 @@ impl CommandResult {
     pub fn code(&self) -> i32 {
         self.output.status.code().unwrap()
     }
+
+    #[allow(dead_code)]
+    pub fn result(self) -> Result<Self, Self> {
+        if self.is_success() {
+            Ok(self)
+        } else {
+            Err(self)
+        }
+    }
```

Comment:
I've seen a few lines like these in the last few days. Maybe it is time to add the glorious [boolinator](https://crates.io/crates/boolinator) as a dependency?